src/main/java/io/stargate/sgv2/jsonapi/service/shredding/collections/DocumentShredder.java

```diff
@@ -241,126 +246,6 @@ private static JsonNode wrapExtensionType(
     return jnf.objectNode().put(etype.encodedName(), value.toString());
   }
 
-  /**
```

Comment:
Refactoring: moved inlined traverse methods to a new helper class without logic changes.